e2e/startupSmoke.spec.ts

```diff
@@ -1,8 +1,72 @@
-import { test, expect } from '@playwright/test';
+import { test, expect, Page, type Locator, BrowserContext } from '@playwright/test';
 
-test('has title', async ({ page }) => {
-    await page.goto('/');
+class DiveInfoPage {
+    private readonly titleSelector = '#dive-info-title';
+    private readonly timeLabelSelector = '#total-dive-time';
+    private readonly timeValueSelector = '#total-dive-time-value';
+    private readonly waypointTableSelector = '#dive-waypoints-table';
+    private readonly waypointRowsSelector = '#dive-waypoints-table tbody tr';
 
-    await expect(page).toHaveTitle(/Dugong Gas Planner/);
-});
+    constructor(private page: Page) {}
+
+    async navigate() {
+        await this.page.goto('http://localhost:4200/');
```

Comment:
The base address is already provided in the playwright config.
We dont need to use absolute path, see the original code.